src/Nri/Ui/TextArea/V5.elm

```diff
@@ -42,7 +42,7 @@ module Nri.Ui.TextArea.V5 exposing
   - Adds field for onBlur
 
 
-## The Nri styleguide-specified textarea with overlapping label
+## The Nri component-catalog-specified textarea with overlapping label
```

Comment:
```suggestion
## The Nri Component-Catalog-specified textarea with overlapping label
```
(I don't actually entirely know what this means, so no sure if "Nri" is needed or if it should be "NRI"? Just a flag - your call!)